### DISCUSSION TITLE
[Bug] The tailwind.config.ts file is not working

### DISCUSSION THREAD

**Discussion by xiaohaoo** [tailwindlabs/tailwindcss#11342] [Help]
**What version of Tailwind CSS are you using?**

v3.3.2

**What build tool (or framework if it abstracts the build tool) are you using?**

create-react-app v5.0.1

**What version of Node.js are you using?**

v18.12.1

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS, Windows

**Reproduction URL**

[https://codesandbox.io/s/interesting-fire-roke92](https://codesandbox.io/s/interesting-fire-roke92)

**Describe your issue**

When I use **tailwind.config.ts** configuration, it will not have any effect.

**Comment by adamwathan** [comment]
Hey! Multiple issues here:

1. You had Tailwind CSS v3.2 installed, TypeScript support was added in v3.3.
2. Tailwind is a build tool so it doesn't run properly in CodeSandbox's browser-only sandboxes, need to switch to a Cloud instance.

I can never get CodeSandbox to do anything I want otherwise I'd fork it and post a link here, but using the latest version of Tailwind should solve it in a real project.

I'd also recommend not using Create React App, it is effectively deprecated by the React team, not actively maintained, and doesn't allow customizing your PostCSS configuration which makes using it with Tailwind very painful.

**Reply by xiaohaoo** [reply]
Thank you. If don't recommend using the Create React App, which one do you recommend？

**Reply by Bay102** [reply]
Vite is the new recommended build by React

**Reply by xiaohaoo** [reply]
Thank you very much 😁
